test/common/testinfra.py

```diff
@@ -149,6 +149,30 @@
     'TEST_DIR',
 )
 
+def determine_github_base():
+    # pick a base
+    # see if we have a git checkout - it can be in https or ssh format
+    formats = [
+        re.compile("""origin\thttps:\/\/github\.com\/(.*) \(fetch\)"""),
+        re.compile("""origin\tgit@github.com:(.*)\.git \(fetch\)""")
+        ]
+
+    try:
+        remote_output = subprocess.Popen(["git", "remote", "-v"], stdout=subprocess.PIPE, cwd=TEST_DIR).communicate()[0]
```

Comment:
> Why isn't this: git config --local --get branch.master.remote ?

On my system that command only outputs `origin`. What am I supposed to see?

what we are interested in is the GitHub base location, e.g. `cockpit-project/cockpit`.